app/policies/keyword_policy.rb

```diff
@@ -15,6 +15,10 @@ def create?
     user_valid?
   end
 
+  def show?
+    user_valid?
+  end
+
   private
 
   def user_valid?
```

Comment:
`user && !user.id.nil? && !user.email.nil?` 

Not sure when I missed this one 🙈 

Can we improve it a little bit? 

`user&.id.present? && user&.email.present?`